runner/ansible/roles/checks/1.3.1/defaults/main.yml

```diff
@@ -28,7 +28,7 @@ remediation: |
      ```crm configure property maintenance-mode=false```
 
   ## References
-  - T.B.D.
+  - https://documentation.suse.com/sle-ha/15-SP3:/html/SLE-HA-all/cha-ha-storage-protect.html
```

Comment:
This link doesn't work. Maybe some typo?

Reply:
Ok, I see that there is `:` in the URL. Can you fix it @diegoakechi ?

Reply:
done